### PR TITLE
Updated German Translations

### PR DESCRIPTION
**Pull request by ninelore**
Corrected weird Grammar and added missing Translations

### CODE REVIEW

File: i18n/de.json [hunk 1]
```diff
@@ -83,22 +83,22 @@
   "app.main.needreloadstash": "Lade World of Warcraft's UI neu um neue Updates zu laden",
   "app.main.needrestart": "Starte World of Warcraft's UI neu um die Updates in WeakAuras' Optionen zu sehen",
   "app.main.nothingToFetch": "Keine Updates verfügbar",
-  "app.main.settingswiped": "Companion's settings have been reset!",
+  "app.main.settingswiped": "Die Einstellungen der Companion wurden zurückgesetzt!",
```
Comment (Stanzilla) [inline i18n/de.json]: *des

File: i18n/de.json [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
-  "app.aura.versionskip": "version skipped",
+  "app.aura.versionskip": "Version übersprungen",
   "app.config.account.notvalidtooltip": "Wir konnten keine WeakAura Dateien auf diesem Account finden. Hast du das AddOn installiert?",
   "app.config.account.selectaccounttooltip": "Wähle einen Account",
   "app.config.autoStart": "Client starten, wenn mein Computer gestartet wird",
   "app.config.autoupdater": "Auto-Update",
-  "app.config.autoupdater.autoupdate": "Install Companion updates automatically",
-  "app.config.autoupdater.beta": "Use Companion Beta channel",
-  "app.config.autoupdater.check": "Check for COmpanion Updates",
+  "app.config.autoupdater.autoupdate": "Installiere Companion Updates Automatisch",
```
Comment (Stanzilla) [inline i18n/de.json]: *updates automatisch

Comment (ninelore) [reply i18n/de.json]: "automatisch" is now lowercase while "(Die) Updates" is a noun

File: i18n/de.json (hunk 1 above)
Comment (ninelore) [reply i18n/de.json]: fixed

File: i18n/de.json [hunk 3]
```diff
@@ -30,27 +30,27 @@
...
-  "app.config.wagoApiKey": "Set Wago API Key (optional)",
+  "app.config.wagoApiKey": "Setze Wago API Schlüssel (optional)",
   "app.config.wagoSettings": "Wago Einstellungen",
-  "app.dropdown.placeholder": "Select",
+  "app.dropdown.placeholder": "Auswählen",
   "app.fileselect.backupfolder": "Backup Ordner",
   "app.fileselect.wowfolder": "World of Warcraft Ordner",
-  "app.footer.browsewago": "Browse Wago for more auras!",
-  "app.footer.getweakauras": "Get WeakAuras!",
+  "app.footer.browsewago": "Durchsuche Wago für mehr Auras!",
```
Comment (Stanzilla) [inline i18n/de.json]: *Auren :D

**Comment by Stanzilla** [comment]
Danke dir!

File: i18n/de.json (hunk 3 above)
Comment (ninelore) [reply i18n/de.json]: 4 eyes see more then 2 :wink: 
also fixed

**Pull request merged**